lldb/packages/Python/lldbsuite/test/lldbtest.py

```diff
@@ -1407,10 +1383,10 @@ def isLoongArch(self):
         return arch in ["loongarch64", "loongarch32"]
 
     def isLoongArchLSX(self):
-        return self.isLoongArch() and "lsx" in self.getCPUInfo()
+        return self.isLoongArch() and self.isSupported(cpu_feature.LSX)
 
     def isLoongArchLASX(self):
-        return self.isLoongArch() and "lasx" in self.getCPUInfo()
+        return self.isLoongArch() and self.isSupported(cpu_feature.LASX)
```

Comment:
Some of these is architecture and has feature should probably be broken up again, and the tests annotated with two requirements instead.

Something for the future though not here.